aiida/backends/djsite/db/migrations/0020_provenance_redesign.py

```diff
@@ -75,7 +75,6 @@ def detect_unexpected_links(apps, schema_editor):
 
 def reverse_code(apps, schema_editor):
     """Reversing the inference of the process type is not possible and not necessary."""
-    pass
```

Comment:
why should this be deleted?

Reply:
Because it is not necessary if there is a docstring

Reply:
I see